clusters/plugin-health-scoring.yaml

```diff
@@ -0,0 +1,10 @@
+  - name: plugin-health-scoring
+    namespace: plugin-health-scoring
+    chart: jenkins-infra/plugin-health-scoring
+    version: 2.1.5
+    needs:
+      - public-nginx-ingress/public-nginx-ingress # Required to expose the service
+    values:
+      - "../config/plugin-health-scoring.yaml"
+    secrets:
+      - "../secrets/config/plugin-health-scoring/secrets.yaml"
```

Comment:
Looks like an error (`git add .` is bad :troll: )

Reply:
That was a `git add clusters`, completely unrelated! 😛 

Good catch, that's from a POC trying to allow external contributor to be in charge of dependency PR review for specific services